ultraplot/figure.py

```diff
@@ -816,14 +816,9 @@ def _context_authorized(self):
     @staticmethod
     def _parse_backend(backend=None, basemap=None):
         """
-        Handle deprication of basemap and cartopy package.
+        Handle potential deprication of basemap and cartopy package.
         """
-        if basemap is not None:
-            backend = ("cartopy", "basemap")[bool(basemap)]
-            warnings._warn_ultraplot(
-                f"The 'basemap' keyword was deprecated in version 0.10.0 and will be "
-                f"removed in a future release. Please use backend={backend!r} instead."
-            )
+        # Basemap is currently being developed again so are removing the deprecation warning
```

Comment:
This is not right. This deprecation warning was about how to specify the backend (via a string) and does not depend on whether or not basemap is being developed. We need to keep this code and force folks to write `backend='basemap'` if they want basemap.

Reply:
The backend is not getting removed imo. At the time Luke developed this basemap was EOL.